### PR TITLE
Update README to correct preference paths

### PR DESCRIPTION
**Pull request by jamescmartinez**
The paths to these settings were incorrect. Fixed! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ it is unchecked. At this point your IDE will work with its regular keyboard
 shortcuts.
 
 Keyboard shortcut conflicts between the Vim emulation and the IDE can be
-resolved via "File | Settings | Vim Emulation", "File | Settings | Keymap"
+resolved via "Preferences | Other Settings | Vim Emulation", "Preferences | Keymap"
```
Comment (vlasovskikh) [inline README.md]: @jamescmartinez Thanks for your pull request. Actually it's File | Settings | ... on Linux and Windows and Preferences | ... on macOS. Could you please mention macOS specific names separately?

**Comment by jamescmartinez** [comment]
@vlasovskikh Updated. Sorry for the delay, I didn't see this until now.

**Comment by jamescmartinez** [comment]
Bump bump. @vlasovskikh  :)

**Comment by vlasovskikh** [comment]
@jamescmartinez Thanks! :)

**Pull request merged**